hassio/system/hassio-host-info.html

```diff
@@ -76,13 +72,22 @@
             path="hassio/host/shutdown"
           >Shutdown</ha-call-api-button>
         </template>
+        <template is='dom-if' if='[[computeUpdateAvailable(data)]]'>
+          <ha-call-api-button
+            hass='[[hass]]'
+            path="hassio/host/update"
+          >Update</ha-call-api-button>
+        </template>
+        <paper-button
```

Comment:
We shouldn't mix actions that "show info" or "mutate state" (ie shutdown/reboot). Maybe we can move the mutating actions to an overflow menu in the top right of a card? Maybe there is a single button that shows a dialog with advanced actions?

Reply:
Or move the 'show hardware' button to within the card.
Else you've got tot change the other cards with mutating actions too.

I think it could be a 'more info' button. Which shows this info, but can later maybe also other info like uptime, memory usage and other information.

Reply:
I also prefer to move the "show hardware" button as you only need it if you add a z-wave stick or want to configure add-ons.